documentation/sign-in-canada.md

```diff
@@ -0,0 +1,36 @@
+# Sign In Canada
+
+## Local setup
+
+### Environment variables
+
+#### `api/.env`
+
+- Comment out block of variables 'for mock oauth testing'
+- Uncomment block of variables 'for SiC testing'
+- Change `OAUTH_ACR_VALUES=gckeymfa` to `OAUTH_ACR_VALUES=gckey`
+- Add values for `OAUTH_API_CLIENT_ID` and `OAUTH_API_CLIENT_SECRET` (these can be obtained from @tristan-orourke)
+
+#### `apps/web/.env`
+
+- Comment out block of variables 'Mock auth endpoint'
+- Uncomment block of variables 'SiC endpoint (will break e2e tests)'
+- Change `OAUTH_ACR_VALUES=gckeymfa` to `OAUTH_ACR_VALUES=gckey`
+- Run `npm run dev` to update config values
+
+### Account creation
+
+1. Navigate to [Sign in using GCkey](http://localhost:8000/en/login-info)
+2. Click the _Continue to GCKey and sign in_ button
+3. Sign in or Sign up for an account
+4. Start your account at the Create an account page
+5. Copy the email address entered in Create an account page form for later
+6. Run `php artisan tinker`
```

Comment:
Maybe add "... from the api directory.

Reply:
Updated in 2a32c3a7aed3c4fff764b685e254dc1f83ba9130.